backend/src/main/java/org/backend/ridinglesson/RidinglessonController.java

```diff
@@ -0,0 +1,37 @@
+package org.backend.ridinglesson;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/ridinglessons")
+
+public class RidinglessonController {
+    private final RidinglessonService ridinglessonService;
+    private final RidinglessonRepo ridinglessonRepo;
+
+    @GetMapping
+    public List<RidinglessonDTO> getAllLessons() {
+        List<Ridinglesson> ridinglessons = ridinglessonService.findAllLessons();
+        return ridinglessons.stream()
+                .map(ridinglesson -> new RidinglessonDTO(ridinglesson.id(), ridinglesson.ridinginstructor(), ridinglesson.ridingtype(), ridinglesson.horse(),
+                        ridinglesson.Date(), ridinglesson.Time(), ridinglesson.status()))
+                .toList();
+    }
+
+    @GetMapping("id")
```

Comment:
ich weiß nicht, ob es auch ohne geht, aber bis jetzt haben wir immer geschweifte Klammern um id gesetzt und ein / davor gemacht
```suggestion
    @GetMapping("/{id}")
```